note.txt

```diff
@@ -0,0 +1,4 @@
+//provider set on index.js
```

Comment:
Can you remove this file and keep it local?

Reply:
note.txt was my own reminder sheet that I forgot to delete.file deleted.